README.md

```diff
@@ -6,13 +6,34 @@
 
 ![Barista logo](art/barista-logo.png)
```

Comment:
Maybe we should put a quick table of contents as in https://github.com/SchibstedSpain/Parallax-Layer-Layout

Reply:
Personally, I never used those quick tables... I think that they break what I expect from a readme. I think we should focus in making the readme shorter, moving the really technical things to other files. Then, we wouldn't need that quick table 😅

Reply:
Other files, like javadocs? 😏  🤡 

I can agree we have too much code in the Readme, we don't need to showcase the whole API...
Is this a blocking thing? Or can we improve it in the future?

Reply:
For me, step by step. Finish this PR as soon as possible because it will improve the readme, and then, move to the next thing. Probably we should talk with new Barista users about what they think about the readme before doing a big change on it.